tools/RAiDER/models/allowed.py

```diff
@@ -11,7 +11,8 @@ def AllowedModels():
       'WRF',
       'HRRR',
       'GMAO',
-      'HDF5']
+      'HDF5'
+      ]
```

Comment:
IMO the indentation should be fixed so it's a multiple of 4

Reply:
Fixed in newest version.